body.tex

```diff
@@ -0,0 +1,440 @@
+ensuring in this way that each issue is fully documented and self consistent\footnote{process to be automated}.
+
+
+\subsection{Release Note} \label{sec:relnote}
+
+The release note documents the content of a release.
+
+The following information should be  provided:
+\begin{itemize}
+\item Installation instructions. This is usually a manually written set of instructions.
+\item Narrative section describing the content of the release, this summary should be written by a person not auto generated.
+The T/CAM is responsible for making sure this is provided.
+\item List of jira issues included in a release. This information can be extracted from Github. Completed epics will be highlighted while other issues will  listed to be comprehensive.
+\item Technical information like the GitHub tag, dependencies,  etc. can be extracted automatically from Github or other tools.
+\end{itemize}
+
+
+\newpage
+\section{Software Release Procedure} \label{sec:releaseprocedure}
+
+This release procedure has been derived from the \textit{Stack release playbook} \citeds{SQR-016}.
+
+\subsection{Development} \label{sec:dev}
+
+Development activities are not part of the release process, but are the starting point for a stable and reliable master branch in all Github software packages.
+The Developer Guide lays out the guidelines and process. All changes are done on ticket branches and reviewed using the Pull Request mechanism before merging to master.
+
+Each time a change is merged into master, the following activities should be performed:
+
+\begin{itemize}
+\item continuous integration build of the Github software package (SW product)
+\item if unit tests pass, generate binary packages
+\item build downstream dependencies: CI build on SW products that depend from the newly build SW product.
+\end{itemize}
+
+At the moment, continuous integration is done on a ticket branch prior to merging changes to master. Binary packages (EUPS) and docker images for distribution are made available with the nightly and weekly builds (see next section \ref{sec:weekly}).
+
+
+\subsection{Nightly and Weekly builds} \label{sec:weekly}
+
+Nightly and weekly builds are  useful for a number of reasons such as :
+\begin{itemize}
+	\item Maintaining a healthy code base.
+	\item Forcing us to maintains build scripts.
+	\item Finding breaking changes which got checked in and passed CI.
+	\item A starting point for a development activity.
+	\item A starting point for a release.
+\end{itemize}
+
+Nightly builds do not generate tags in the Github repositories.
+When a weekly build is done, a corresponding tag in the Github repository is created.
+Though weekly builds are considered releases in Github, from a release management point of view they are not
+releases - they are reference builds and, when they meet the criteria in \ref{sec:firstrc}, release candidate starting points.
```

Comment:
Are nightly & weekly builds different from simply periodically dropping a tag on the the Jenkins-on-`master` process described above?

Reply:
This is not relevant from a release process point of view. However, the only difference that I am able to spot is that weekly builds trigger the doc build, the nightly don't.